src/EditorFeatures/CSharpTest/Diagnostics/HideBase/HideBaseTests.cs

```diff
@@ -93,5 +93,29 @@ class App : Application
     public new int Test;
 }");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddNew)]
```

Comment:
Hi!  Thanks for the contribution!  Please add a [WorkItem()] attribute here that links to the ithub bug.  thanks!